src/main/java/uk/gov/hmcts/reform/sandl/snlevents/config/RestExceptionHandler.java

```diff
@@ -2,12 +2,19 @@
 
```

Comment:
I'm worried that this class is not tested - a functional or unit.

Reply:
The test is added now.